index.html

```diff
@@ -1,12 +1,454 @@
+                    <div class="form-group">
+                        <div class="form-input">
+                            <select id="country" name="country" class="form-input__select">
+                                <option value="australia">Australia</option>
+                                <option value="canada">Canada</option>
+                                <option value="usa">USA</option>
+                            </select>
+                        </div>
+                        <div class="form-input">
+                            <select id="country" name="country" class="form-input__select">
+                                <option value="australia">Australia</option>
+                                <option value="canada">Canada</option>
+                                <option value="usa">USA</option>
+                            </select>
+                        </div>
+                    </div>
+
+                    <div>
+                        <div class="col-75">
+                            <textarea id="subject" class="form-text" name="subject" placeholder="Write something.."
+                                style="height:140px"></textarea>
+                        </div>
+                    </div>
+                    <div>
+                        <div class="btn btn--colored">Get quotes now</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- ============================================================================================ -->
+        <div class="wrapper">
+            <div class="sub-footer">
+                <div class="sub-footer__content">
+                    <h3 class="title">Consulting Agent For Your Business</h3>
+                    <p class="sub-title">The quick fox jumps over the lazy dog</p>
+                </div>
+                <div class="btn btn--orange btn--sm">Contact Us</div>
+            </div>
+        </div>
+        <!-- ========================================================================================== -->
+        <div class="footer">
+            <div class="footer__content">
+                <div>
+                    <h2 class="logo--white">KMarketer</h2>
+                </div>
+                <div class="">
+                    <box-icon color="#335BF5" type='logo' name='facebook-square'></box-icon>
+                    <box-icon color="#E51F5A" type='logo' color="#1BA0AB" name='instagram'></box-icon>
+                    <box-icon color="#21A6DF" type='logo' color="#1BA0AB" name='twitter'></box-icon>
+                    <box-icon color="red" type='logo' name='youtube'></box-icon>
+                </div>
+            </div>
+        </div>
+        <!-- =================================================================================================== -->
+        <div class="footer footer--big footer--start-placed-content">
+            <div class="footer__content footer-content--unbordered">
```

Comment:
`footer-content--unbordered` is included without it's default Element `footer-content`.